src/inspector/internal/inspectorcontroller.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "inspectorcontroller.h"
+#include "inspector/models/popups/abstractelementpopupmodel.h"
```

Comment:
Let's add a newline and remove `inspector/`
```suggestion

#include "models/popups/abstractelementpopupmodel.h"
```

Reply:
It's giving me preprocessor issues when I remove `inspector\` should I ignore it?

Reply:
If you mean that Qt Creator displays errors in the code editor view, that is to be expected; because of _unity builds_, Qt Creator's code model does not have information about how to compile the individual cpp files, because during a real build those unity files would be compiled instead. This means that the code model does not know where to look for included files, and thus displays useless errors/warnings. You can ignore those, the only thing that really matters is what the compiler tells you in "Compile output".